### PR TITLE
upgrade alibaba-cloud-sdk-go for #4715

### LINKED ISSUE #4715: Image scan has detect CVE-2021-3538
Hi, trivy image scanner has detect 1 critical vulnerability in the latest version:

```
trivy image k8s.gcr.io/autoscaling/cluster-autoscaler:v1.23.0
2022-02-25T10:24:50.270+0100	INFO	Detected OS: debian
2022-02-25T10:24:50.270+0100	INFO	Detecting Debian vulnerabilities...
2022-02-25T10:24:50.270+0100	INFO	Number of language-specific files: 1
2022-02-25T10:24:50.270+0100	INFO	Detecting gobinary vulnerabilities...

k8s.gcr.io/autoscaling/cluster-autoscaler:v1.23.0 (debian 11.2)
===============================================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)


cluster-autoscaler (gobinary)
=============================
Total: 2 (UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 0, CRITICAL: 1)

+---------------------------+------------------+----------+-------------------+-------------------------------------+--------------------------------------+
|          LIBRARY          | VULNERABILITY ID | SEVERITY | INSTALLED VERSION |            FIXED VERSION            |                TITLE                 |
+---------------------------+------------------+----------+-------------------+-------------------------------------+--------------------------------------+
| github.com/satori/go.uuid | CVE-2021-3538    | CRITICAL | v1.2.0            | 1.2.1-0.20181016170032-d91630c85102 | satori/go.uuid: predictable UUIDs    |
|                           |                  |          |                   |                                     | generated via insecure randomness    |
|                           |                  |          |                   |                                     | -->avd.aquasec.com/nvd/cve-2021-3538 |
+---------------------------+------------------+----------+-------------------+-------------------------------------+--------------------------------------+
| k8s.io/kubernetes         | CVE-2020-8554    | MEDIUM   | v1.23.0           |                                     | kubernetes: MITM using               |
|                           |                  |          |                   |                                     | LoadBalancer or ExternalIPs          |
|                           |                  |          |                   |                                     | -->avd.aquasec.com/nvd/cve-2020-8554 |
+---------------------------+------------------+----------+-------------------+-------------------------------------+--------------------------------------+

```

Could you please fix this?

**Which component are you using?**:

cluster-autoscaler

**What version of the component are you using?**:

Component version:
image-tag: v1.23.0
Chart-Version: 9.15.0

**What k8s version are you using (`kubectl version`)?**:

AWS EKS 1.21

**What environment is this in?**:

AWS EKS

**What did you expect to happen?**:

No critical vulnerabilites

**What happened instead?**:

1 critical vulnerability

**How to reproduce it (as minimally and precisely as possible)**:

`trivy image k8s.gcr.io/autoscaling/cluster-autoscaler:v1.23.0`

### PR DESCRIPTION
**Pull request by IrisIris**
#### Which component this PR applies to?
alicloud provider

#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
Fixes #4715 

#### Which issue(s) this PR fixes:
Fixes #4715 

#### Does this PR introduce a user-facing change?
NONE

### CODE REVIEW

File: cluster-autoscaler/go.sum [hunk 1]
```diff
@@ -103,6 +103,8 @@ github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751/go.mod h1:LOuy
 github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRFXyAe+OPACYpWeL0wqObRcbAqCMya13uyzqw0=
 github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4/go.mod h1:ybxpYRFXyAe+OPACYpWeL0wqObRcbAqCMya13uyzqw0=
 github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d/go.mod h1:rBZYJk541a8SKzHPHnH3zbiI+7dagKZ0cgpgrD7Fyho=
+github.com/aliyun/alibaba-cloud-sdk-go v1.61.62 h1:APZ+/l4TsHbFx6zZnXJE2QCXs3IChVmcI5GKwZnnmaw=
```
Comment (mwielgus) [inline cluster-autoscaler/go.sum:106]: The purpose of embedding sdk in cloud provider folder is to not add cloud provider specific external dependencies to CA. Please remove these lines.

**Review by mwielgus — CHANGES_REQUESTED** [review]

File: cluster-autoscaler/go.sum (hunk 1 above)
Comment (IrisIris) [reply cluster-autoscaler/go.sum]: Thanks for reviewing! The new alibaba sdk  need the dependency,  gopkg.in/ini.v1 which  is not in vendor yet. Shoud I keep the dependency information in go.sum, go.mod and vendor/modules.txt or remove it?

Comment (mwielgus) [reply cluster-autoscaler/go.sum]: I'm sorry but with 30 cloud providers around, to keep our integrity, we cannot make exceptions. There should be no additional dependencies added for any of the cloud providers, neither in go.mod or vendor, no matter how harmless the dependency may look like.

**Comment by x13n** [comment]
This looks stale, so closing. Please reopen if still relevant.

/close

**Pull request closed without merging**